test/k8sT/Chaos.go

```diff
@@ -34,7 +34,13 @@ var _ = Describe("K8sChaosTest", func() {
 	BeforeAll(func() {
 		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
 
-		err := kubectl.CiliumInstall(helpers.CiliumDefaultDSPatch, helpers.CiliumConfigMapPatch)
+		_ = kubectl.Apply(helpers.DNSDeployment())
```

Comment:
Why we don't check the status?

Reply:
I don't remember but I think it was failing to deploy kube-dns because it was already deployed. I can try to add the error again and check.

Reply:
@eloycoto it fails https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/6783/testReport/junit/k8s-1/11/K8sChaosTest_Endpoint_can_still_connect_while_Cilium_is_not_running/